src/main/drivers/pwm_output.c

```diff
@@ -779,4 +773,14 @@ void beeperPwmInit(const ioTag_t tag, uint16_t frequency)
     }
 }
 #endif // USE_BEEPER
+#endif //USE_PWM_OUTPUT
+
+bool isMotorProtocolDshot(void)
```

Comment:
This should be conditional to `USE_PWM_OUTPUT`. If there is no PWM output, the concept of motor protocol is not defined, so places calling this in this case need to be fixed.

Reply:
I agree, there is a code smell with `isMotorProtocolDshot` for sure.   Seems that the system needs to know if the motor protocol is dshot for a number of reasons when I had a brief look...

This PR doesn't attempt to fix that, only to allow the compilation so that it can be refactored in future.  I don't have time for that task while I have all my other PR's to attend to.  Suggest merging this as-is and creating a ticket to refactor `isMotorProtocolDshot` usage.

Reply:
fixing....